README.md

```diff
@@ -73,6 +73,8 @@ Environment variables **not ready yet**:
 * Email accounts configuration
   * `EMAIL_ACCOUNT_ADDRESS_ACCOUNT`
   * `EMAIL_ACCOUNT_PASSWORD`
+* Home page configuration
+  * `HOME_PAGE`
```

Comment:
The home page configuration is part of the website configuration. They should be grouped together.